internal/apiresource/rolebinding.go

```diff
@@ -67,13 +67,7 @@ func (*RoleBinding) createNewResource(irrolebinding irtypes.RoleBinding) *rbac.R
 	return roleBinding
 }
 
-// ConvertToClusterSupportedKinds converts the object to supported types if possible.
-func (rb *RoleBinding) ConvertToClusterSupportedKinds(obj runtime.Object, supportedKinds []string, otherobjs []runtime.Object, _ irtypes.EnhancedIR) ([]runtime.Object, bool) {
-	supKinds := rb.GetSupportedKinds()
-	for _, supKind := range supKinds {
-		if common.IsStringPresent(supportedKinds, supKind) {
-			return []runtime.Object{obj}, true
-		}
-	}
-	return nil, false
+// convertToClusterSupportedKinds converts the object to supported types if possible.
+func (rb *RoleBinding) convertToClusterSupportedKinds(obj runtime.Object, supportedKinds []string, otherobjs []runtime.Object, _ irtypes.EnhancedIR) ([]runtime.Object, bool) {
```

Comment:
why are these checks removed?

Reply:
this function is to moving between kind concepts if there is a similar concept supported in the cluster. In case there is no equivalent concept and the resource is required, it is better to retain the object. It is for the final writer to decide whether to serialize it or not.